### PR TITLE
bump to 1.0.0rc15

### PR DESCRIPTION
**Pull request by blankdots**
### Description

update docker image with rust in build, needed by cryptography package https://cryptography.io/en/latest/installation.html#rust
There was an `OSError` exception in health endpoint when other endpoints are down so decided to catch them all (shameless Pokemon reference)

### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

### Changes Made

1. breaking changes in cryptography require rust > 1.45 which is in alpine 3.13
2. updated npm packages
3. catch all exceptions in health call

### Testing

- [x] Tests do not apply

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -28,11 +28,14 @@ COPY setup.py /root/swift_ui/setup.py
...
+    && rm -rf /var/cache/apk/*
+
 RUN pip install --upgrade pip && \
     pip install -r /root/swift_ui/requirements.txt && \
     pip install /root/swift_ui
 
-FROM python:3.8-alpine3.12
+FROM python:3.8-alpine3.13
 
 RUN apk add --no-cache --update bash
```
Comment (vvaltia) [inline Dockerfile:41]: Not required for right now, but would make more sense to have just one docker image. If the files differ, they could be added as configmaps instead.

File: swift_browser_ui/health.py [hunk 2]
```diff
@@ -120,6 +132,12 @@ async def get_upload_runner(
         services["swiftui-upload-runner"] = {
             "status": "Error"
         }
+    except Exception as e:
+        request.app["Log"].debug("Poll swift-x-account-sharing failed")
+        request.app["Log"].info(f"reason: {e}")
+        services["swift-sharing-request"] = {
+            "status": "Error"
+        }
```
Comment (vvaltia) [inline swift_browser_ui/health.py:140]: Imo this would make a good function, since the exact same exception is here multiple times

**Review by vvaltia — DISMISSED** [review]

File: swift_browser_ui/health.py (hunk 2 above)
Comment (blankdots) [reply swift_browser_ui/health.py:140]: fair enough will add that.

**Review by teemukataja — DISMISSED** [review]

File: Dockerfile (hunk 1 above)
Comment (blankdots) [reply Dockerfile:41]: for this dockerfile `BACKEND` builds the packages while the layer you pointed has the binaries
If we take into consideration other swift-ui services sure, that could be a way.

**Pull request merged**